lib/modules/platform/bitbucket/index.md

```diff
@@ -1,5 +1,28 @@
 # Bitbucket Cloud
 
+## Authentication
+
+First, [create an app password](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/) for the bot account.
+Give the bot app password the following permission scopes:
+
+- [`account`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#account) (Account: Read)
+- [`team`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#team) (Workspace membership: Read)
+- [`issue:write`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#issue-write) (Issues: Write)
+- [`pullrequest:write`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#pullrequest-write) (Pull requests: Write)
```

Comment:
How about we convert the list to a table and get rid of the `(Account: Read)` stuff? Example:

| Permission | Scope |
| --- | --- |
| [`account`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#account) | `read` |